tests/plugins/test_accounts.py

```diff
@@ -319,3 +321,59 @@ def test_fallsback_on_basicauth(self):
     def test_raise_configuration_if_wrong_error(self):
         with self.assertRaises(ConfigurationError):
             self.make_app({'multiauth.policies': 'basicauth account'})
+
+
+class CreateUserTest(unittest.TestCase):
+    def setUp(self):
+        self.registry = mock.MagicMock()
+        self.registry.settings = {
+            "includes": "kinto.plugins.accounts",
+        }
+
+    def test_create_user_in_read_only_display_an_error(self):
+        with mock.patch('kinto.plugins.accounts.scripts.logger') as mocked:
+            self.registry.settings['readonly'] = 'true'
+            code = scripts.create_user({'registry': self.registry})
+            assert code == 51
+            mocked.error.assert_called_once_with('Cannot create a user with '
+                                                 'a readonly server.')
+
+    def test_create_user_when_not_included_display_an_error(self):
+        with mock.patch('kinto.plugins.accounts.scripts.logger') as mocked:
+            self.registry.settings['includes'] = ''
+            code = scripts.create_user({'registry': self.registry})
+            assert code == 52
+            mocked.error.assert_called_once_with('Cannot create a user when the accounts '
+                                                 'plugin is not installed.')
+
+    def test_create_user_with_an_invalid_username_and_password_confirmation(self):
+        with mock.patch('kinto.plugins.accounts.scripts.input', side_effect=["&zert", "username"]):
+            with mock.patch('kinto.plugins.accounts.scripts.getpass.getpass', side_effect=[
+                    "password", "p4ssw0rd", "password", "password"]):
+                code = scripts.create_user({'registry': self.registry}, None, None)
+                self.registry.storage.create.assert_called_once()
+                self.registry.permission.add_principal_to_ace.assert_called_with(
+                    "/accounts/username", "write", "account:username")
+                assert code == 0
```

Comment:
In this test, the titlle suggests that it will fail to create it (compared to the next test below)